uiowa/uiowa
# UI Press sale updates

Resolves #5468 




# How to test


`git checkout uipress-sale-update && git pull`
`blt ds --site=uipress.uiowa.edu`
`drush @uipress.local uli /books/american-pipe-dream`
^ The paperback should show up with a sale price
